.github/workflows/integration.yaml

```
@@ -58,5 +57,5 @@ jobs:
       - name: Integration test - small dataset
         working-directory: .tests/integration
         run: |
-          mamba install -c conda-forge -c bioconda snakemake singularity=3.8.6
+          mamba install -c conda-forge -c bioconda snakemake=6.10.0 singularity=3.8.6
```

```suggestion
          mamba install -c conda-forge -c bioconda singularity=3.8.6
```

didn't we talk about trying to use the snakemake version installed via pip?

---

change it